### PR TITLE
Updates hydrogen-production-presentation

### LINKED ISSUE #57: update hydrogen-production-pres
This issue can be closed when a PR updates hydrogen-production-pres presentation.

### PR DESCRIPTION
**Pull request by robfairh**
Closes #57 

Updates hydrogen-production-presentation for the 2020 ANS Annual meeting.

### CODE REVIEW

File: hydrogen-production-pres/conclusions.tex [hunk 1]
```diff
@@ -4,8 +4,8 @@
 \begin{itemize}
 	\item The University of Illinois is actively working to reduce GHG emissions on its campus. 
 	\item A few microreactor designs would be able to produce enough hydrogen to meet MTD and UIUC fleet fuel demand.
-	\item An excessive integration of PV to UIUC grid make the duck curve phenomenon likely to occur.
-	\item Nuclear energy and hydrogen production proposes an approach to mitigate the negative implications of the duck curve.
+	\item An excessive integration of PV to UIUC grid will make the duck curve phenomenon likely to occur.
```
Comment (samgdotson) [inline hydrogen-production-pres/conclusions.tex]:
```suggestion
	\item Increased solar penetration worsens the duck curve.
```

Comment (samgdotson) [reply hydrogen-production-pres/conclusions.tex]: Not essential, just a suggestion for clarity.

File: hydrogen-production-pres/conclusions.tex [hunk 2]
```diff
@@ -4,8 +4,8 @@
...
+	\item Nuclear energy and hydrogen production present an approach to mitigate the negative implications of the duck curve.
```
Comment (samgdotson) [inline hydrogen-production-pres/conclusions.tex]: I'd move this conclusion below the "hydrogen introduces..." because it ties everything together.

**Review by samgdotson — APPROVED** [review]
Looks good, there are a couple of wording changes that I'd make but they're non-essential.

File: hydrogen-production-pres/conclusions.tex (hunk 1 above)
Comment (robfairh) [reply hydrogen-production-pres/conclusions.tex]: looks good

File: hydrogen-production-pres/conclusions.tex (hunk 2 above)
Comment (robfairh) [reply hydrogen-production-pres/conclusions.tex]: done

**Comment by samgdotson** [comment]
Awesome, merging. 

Closes #57

**Comment by datw0258** [comment]
This is just a recommendation (because I have have seen both used), but typically in bullet points I have tended to only use a period or some other forms of punctuation if the bullet point is a sentence itself or completes one.

This may just be a personal preference, however.

**Review by datw0258 — COMMENTED** [review]
Looks good otherwise.

**Pull request merged**